### DISCUSSION TITLE
PvP Feedback 2: Electric Boogaloo

### DISCUSSION THREAD

**Discussion by ghost** [FakeFishGames/Barotrauma#14495] [Unstable feedback]
stuff I saw and had thoughts on throughout a 2 hour session

# Outpost Deathmatch

- People rushing to grab weapons means the team that gets guns first tends to have a major advantage- dead players can't loot crates and free points for the victor.
- Alien pistols are absurdly deadly. They do not belong in the more common crates with their EXTREMELY HIGH DPS when dual wielded and the fact they ignore body armor.
- HMGs are useless. They're worse than diving knives: the stun prevents even aiming them and exposes you to attack.
- Grenade launchers win. Every time. If it's grenade launcher vs grenade launcher then whoever clicks first (and has better ping) wins.
- There's too many weapons per crate. Having more powerful weapons available was a leap in the right direction... but the goal was overshot and now the round is saturated with extremely strong gear.
- Cover is a good concept (and works well with 2d combat, as shown by modded gamemodes such as _Middle East PvP_ which make heavy use of it), but it's too short and isn't actually useful. The most dangerous weapons are very accurate or aren't really stopped by cover anyway.

# King of the Hull

- It's really hard to get someone off the beacon once they're on: the first team there has a huge advantage since you need to fight through their sub AND get past their turrets to reach the beacon.

# Other stuff!
- The vest on  "Separatist Engineer Outfit"'s sprite is too muddy, and it's not clear what it's meant to be unless you look particularly close.

**Comment by ubertpendragon** [comment]
Some beacons have invincible walls too so you can't cut through them (if the plasma cutter spawns at all)

**Comment by bananaman6213** [comment]
To add my own thoughts on outpost deathmatch:

- I had success using cover to ragdoll behind and stage ambushes, rather than trying to crouch behind it in a firefight. I think it's honestly in a good place
- Grenade launchers really are terrifying, and if someone lands the first hit on you with one you're usually dead to rights
- The default respawn timer is far too long

- Wiring is very difficult
I want to expand on this point for a bit because I think wiring in this could be fun. One of the crates contains a detonator and UEX or C4, which initially seems useful. My first thought was to make a door bomb with it, but that's not doable without the ability to wire to the door (all outpost devices have locked wiring). The only feasible option would be scrounging for a component and welder to decon into materials for a button, and then finding a wire and screwdriver in a cabinet to wire the button to the detonator. Doing this without getting killed is a tall order, especially for an explosive like UEX which probably won't oneshot, from my experience.

However, if wiring to outpost devices were enabled there would really be nothing stopping you from locking doors, and with doors being invincible that would be a major issue. You could also mess with pumps and flood the entire outpost, which sounds like it would be extremely degenerate.

Maybe if the detonator crates came with a few amenities (screwdriver, wires, misc components to deconstruct), wiring could be made easier? It's hard to say and I have no idea how many players would even really use it anyways.
Arguably it might just be better for the detonator/explosives to be removed.

**Reply by JeroonBosch** [reply]
> Arguably it might just be better for the detonator/explosives to be removed.

I replaced the detonators with the new timed detonator variant instead, which should be much easier to use (without wiring required, but is still optional)

**Comment by Arctodites** [comment]
I've been waiting so long for outpost deathmatch!  It could easily be what attracts a whole new playerbase to the game -- it's really that good.

**---- Wiring: detonators, timers, tripwires**
Wiring has so much potential as a playstyle in outpost deathmatch.  I agree with @bananaman6213 that detonator crates should come with a couple wires, a button, and maybe a multi-tool.  A motion detector would also be nice, but they are persistent items, so maybe there could be a (consumable) laser tripwire instead?  The other common problem with building complex circuitry in a deathmatch is the lack of a dedicated countdown timer component: either detonators should be programmable, or there should be a separate countdown timer with set / reset functionality.

**---- Cover: fixed, movable, and regenerating**
Fixed cover is another great addition to the game maps.  I hope the devs experiment with it further.  Last I checked, human characters are 11 units tall (standing) or 8 units tall (crouching).  The maximum safe hurdle height that characters can comfortably scale is 6 units; anything 9 units or taller is unclimbable, and the maximum incline that characters can walk up is 50 degrees.  So any scalable fixed cover that is vertical (i.e., not ramped as in the _Raate_ map) cannot be taller than 6 units on the side facing the attacker.  Contrastingly, it is possible for the defender's side of the cover to be deepened to 8 units by means of a shallow trench, just so long as there is a small step or ladder to help the defender climb up over the barrier.  This type of redoubt would solve the problem of characters' heads being exposed to gunfire while crouching behind cover.

Movable cover and regenerating fixed cover would also be great to have.  If players could stack heavy metal boxes or blocks of ice, for example, to reinforce defensible positions.  New boxes or blocks of ice could spawn at a cargo door at one end of a moving conveyor belt and despawn when they reach a cargo door at the opposite end.  Conveyor belts -- whether floor conveyors or overhead tracks -- could add a whole new dynamicity to level design.  These machines would be right at home in the outpost tunnels and mines -- a type of environment we've not really had a chance to see fully fleshed out in-game.

Ice stalactites, stalagmites, or cut blocks could be destructible (similar to glass), but instead of breaking permanently, they could refreeze (i.e. regain health) over time.  The crunchy, crackling sound of ice breaking would also be endlessly satisfying.

Another way to make regenerating barriers could be particle shields that allow only one-way travel / missile fire.  The shields could fail when they take too much damage or run out of battery power.

**Reply by ghost** [reply]
> Last I checked, human characters are 11 units tall (standing) or 8 units tall (crouching). The maximum safe hurdle height that characters can comfortably scale is 6 units; anything 9 units or taller is unclimbable, and the maximum incline that characters can walk up is 50 degrees. So any scalable fixed cover that is vertical (i.e., not ramped as in the Raate map) cannot be taller than 6 units on the side facing the attacker.

You can simply have a platform cut through the cover and it can be signifigantly taller since characters can move up the platform and then hurdle the cover from there.

**Comment by ubertpendragon** [comment]
I would like to see mild turret balance adjustments:

Laser:
+ structure damage
+ power consumption
The structure damage is almost nonexistent, a basic coilgun deals 6x the structure damage per second, I suspect anyone picking a sub with pulse laser in PVP will lose. It's also pretty bad in PvE when you want to kill an iceberg.
Power consumption is however nonexistent so you can shoot about 50 consecutive shots at 70% recharge rate before running out.

Coilgun
+ power consumption
Coilgun is generally balanced besides a very low power consumption.

**Comment by JeroonBosch** [comment]
Thanks for the feedback everyone, we're in the process of addressing much of it!
